release-blocking-jobs.md

```diff
@@ -2,56 +2,127 @@
+
+- It provisions clusters via open-source software instead of a hosted service
+<!-- TODO(spiffxp): 
+  - should any allowances be made for conformance?
+-->
+- It finishes a run in 120 minutes or less
+<!-- TODO(spiffxp): 
+  - this should be a percentile, not a max 
+-->
+- It runs at least every 3 hours
+- It passes 75% of all of its runs in the past week (regardless of underlying
+  commit(s))
+- Is capable of passing 3 times in a row against the same commit
+- Is owned by a SIG that is response to addressing failures
+<!-- TODO(spiffxp):
+  - represented by a configured testgrid alert
+  - contact info is also part of job description
+-->
+- Fails for no more than 10 runs in a row
+- Has a documented reason for its inclusion in the release-blocking suite
+<!-- TODO(spiffxp):
+  - represented in description? link to an issue where discussion was had?
+-->
+
+The reasoning for the above is that as we get to the end of a release cycle,
+we are usually waiting on a handful of bug fixes and want to be certain that
+the underlying bug is truly fixed. This means we prefer to see multiple test
+results for the same commit. If we wait for three successful runs that are
+scheduled every three hours, we wait for nine hours.  This effectively gives
+us the chance to make two changes/decisions in a 24 hour period.
+
+The thresholds given are aspirational, and we recognize that not every job may
+be able to strictly adhere to all of these criteria all of the time. Thus we
+consider human review a key component in both the addition and removal of a
+job's release-blocking status.
+
+We intend to drive the criteria to stricter thresholds over time, e.g.,
+- finishing a run in 60min or less
+- running at least every 2 hours
+- passing at least 90% of all runs
+
+## Release-Informing Criteria
+
+Jobs that are considered useful or necessary to inform whether a commit on
+master is ready for release, but that clearly do not meet release-blocking 
+criteria, may be placed on the sig-release-master-informing dashboard. These
+are often jobs that are some combination of slow, flaky, infrequent, or
+difficult to maintain.
+
+These jobs may still block the release, but because they require a lot of
+manual, human interpretation, we choose to move them to a separate dashboard.
+
+<!-- TOOD(spiffxp): implement release-master-informing, populate with e.g.:
+- gce-master-scale-performance
+- gce-cos-master-serial
+- gke-cos-master-serial
+- packages-pushed-master
+- conformance jobs that run less frequently than every 3 hours
+-->
+
+## Release-Blocking Dashboards
+
+<!-- TODO(krzyzacy): use master-blocking dashboard, once we sort out jobs
+                     like scalability -->
 
 We use [1.12 release-blocking dashboard](http://k8s-testgrid.appspot.com/sig-release-1.12-blocking)
-as a source of truth:
+as a source of truth. The tab names are autogenerated based on the job names,
+and have the branch/version in their name. For example, for release x.y:
```

Comment:
I'd like a comment here that this is just the initial set of tests, and that each of these will be evaluated based on the criteria above.  It's really not clear from the description that that's the case, just from your comments.

Reply:
Hmmmm, never mind, you DO have that above.